pkg/providers/cloudstack/cloudstack.go

```diff
@@ -367,6 +419,26 @@ func (p *cloudstackProvider) validateEnv(ctx context.Context) error {
 	return nil
 }
 
+func (p *cloudstackProvider) validateSecretsUnchanged(ctx context.Context, cluster *types.Cluster) error {
+	for _, profile := range p.execConfig.Profiles {
+		secret, err := p.providerKubectlClient.GetSecret(ctx, profile.Name, executables.WithCluster(cluster), executables.WithNamespace(constants.EksaSystemNamespace))
```

Comment:
I think this will fail when the secret doesn't exit, right?
But we don't want to fail in that case, we do allow for new secrets.

Reply:
Got it